lib/read.js

```diff
@@ -63,8 +63,14 @@ function assembleReadQuery(opts) {
 
     var rels = compositionRelationCypherList(comps);
     var start = level ? '__sm_level' + (level - 1) : opts.varName;
+    var dir = (opts.compMapping[level].matches || opts.compMapping[level][rels.replace(/`/g, '')]).direction;
```

Comment:
not quite sure I understand what's going on here. where is `matches` coming from?